### PR TITLE
Add support for custom service labels

### LINKED ISSUE #269: Feature request: Specify custom labels for service
Our proxy auto-discovers services based on labels, and at this point I don't know of a better way to add the necessary labels other than to create the AWX instance, wait for the service to appear, then to patch the labels in. Is there a more elegant way to include custom labels in the service than this, and if not can we add a property for custom labels?

### PR DESCRIPTION
**Pull request by kimbernator**
Fixes: #269

### CODE REVIEW

File: deploy/olm-catalog/awx-operator/manifests/awx-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -381,6 +381,11 @@ spec:
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:advanced
         - urn:alm:descriptor:com.tectonic.ui:hidden
+      - displayName: Tower Service Labels
+        path: tower_service_labels
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:text
```
Comment (tchellomello) [inline deploy/olm-catalog/awx-operator/manifests/awx-operator.clusterserviceversion.yaml:388]: Can you make this field hidden?

**Review by tchellomello — CHANGES_REQUESTED** [review]
@kimbernator Thanks for your contribution.. It looks good just a small change to make the attribute class `hidden` . Thanks!

File: deploy/olm-catalog/awx-operator/manifests/awx-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (kimbernator) [reply deploy/olm-catalog/awx-operator/manifests/awx-operator.clusterserviceversion.yaml:388]: Just updated.

**Comment by kimbernator** [comment]
Requested change has been completed.

**Review by rooftopcellist — APPROVED** [review]

**Review by tchellomello — APPROVED** [review]
:+1:

**Pull request merged**